compiler/src/dotty/tools/dotc/parsing/Parsers.scala

```diff
@@ -3114,12 +3116,13 @@ object Parsers {
               ImportSelector(wildcardSelectorId())
             case GIVEN =>
               val start = in.skipToken()
-              def givenSelector() = atSpan(start) { Ident(nme.EMPTY) }
               if in.token == USCORE then
                 in.nextToken()
-                ImportSelector(givenSelector()) // Let the selector span all of `given _`; needed for -Ytest-pickler
+                ImportSelector(givenSelectorId(start)) // Let the selector span all of `given _`; needed for -Ytest-pickler
```

Comment:
Is this code path still needed for compatibility? If so, should we issue a deprecation warning here?

Reply:
So, far, we were changing syntax without warnings. But maybe we should change that now...

Reply:
I tried to deprecate but that does not work for the current cycle since we still use `given _` in bootstrap code. So we could deprecate at the earliest once 3.0 M1 is released. Or we just rip off the bandaid and disallow it altogether for M2.

Reply:
We can do a bootstrap before M1 is released (and we probably should do one to make sure everything works OK with the new naming scheme)

Reply:
OK. We can deprecate again once that's done